corehq/apps/export/models/new.py

```diff
@@ -230,42 +289,70 @@ def _generate_schema_from_xform(xform, langs, appVersion):
 
         return schema
 
-    @staticmethod
-    def _merge_schema(schema1, schema2):
-        """Merges two ExportDataSchemas together
 
-        :param schema1: The first ExportDataSchema
-        :param schema2: The second ExportDataSchema
-        :returns: The merged ExportDataSchema
-        """
+class CaseExportDataSchema(ExportDataSchema):
 
-        schema = ExportDataSchema()
+    @staticmethod
+    def generate_schema_from_builds(domain, app_id, case_type):
```

Comment:
not sure if this question belongs here but are we going to let people merge case properties across apps? or is that out of scope for now.